lms/static/coffee/spec/instructor_dashboard/send_email_spec.coffee

```diff
@@ -0,0 +1,82 @@
+describe "Bulk Email Queueing", ->
```

Comment:
Uck, Coffee. Given that you've already written this, let's leave it in coffee, but in the future please chat with me about whether or not we can just write it in JS (discussions have gone that route, with the newer code just in JS, though the files still live in the coffee directory).